utils.py

```diff
@@ -0,0 +1,167 @@
+import math
+import random
+
+def n_random_points(n):
+    """
+    :param n:the amount of random points you wish to create
+    :return :the list of n (tuple) points
+    """
+
+    #use list comprehensions to generate n point list
+    points = [(random.randint(0,500), random.randint(0,500)) for x in range(n)]
```

Comment:
This is the line that is causing the error.  Why do you make the assumption that the points are in the range 0,500 and are integers?  I would use `random.rand` and then, if you like, pass in a scaling value or use some other logic to alter the domain.